app/services/karma_service.rb

```diff
@@ -14,22 +14,52 @@ def show_statistic
   def statistic_params
     {
       chat_id: @chat.id,
-      text: statistic_message
+      text: statistic_message,
+      parse_mode: 'HTML'
     }
   end
 
   def statistic_message
+    <<~STRING
+      <b>🏆This is the karma champions list</b>🏆
+      <pre>------------------------------
+      #{user_statistic}
+      ------------------------------
+      </pre>
+    STRING
+  end
+
+  def username(user_name)
+    name = user_name || 'null'
+    until name.length == 20 do name += ' ' end
+    name
+  end
+
+  def user_statistic
     medals = %w[🥇 🥈 🥉]
-    result = '🏆This is the karma champions list🏆'
-    DB.users.each { |user| result << user_statistic(user, medals) }
-    result
+    ''.yield_self do |result|
```

Comment:
You can use `then` instead of `yield_self`. I think it's more readeable

Reply:
Ruby 2.5.0 doesn't have a method `then`